runpod/cli/groups/project/functions.py

```diff
@@ -103,24 +104,38 @@ def launch_project():
         raise ValueError('Project pod already launched. Run "runpod project start" to start.')
 
     print("Launching pod on RunPod...")
-
     environment_variables = {"RUNPOD_PROJECT_ID": config["PROJECT"]["UUID"]}
     for variable in config['project']['env_vars']:
         environment_variables[variable] = config['project']['env_vars'][variable]
-
-    new_pod = create_pod(
-        f'{config["PROJECT"]["Name"]}-dev ({config["PROJECT"]["UUID"]})',
-        config['PROJECT']['BaseImage'],
-        config['PROJECT']['GPU'],
-        gpu_count=int(config['PROJECT']['GPUCount']),
-        support_public_ip=True,
-        ports=f'{config["PROJECT"]["Ports"]}',
-        network_volume_id=f'{config["PROJECT"]["StorageID"]}',
-        volume_mount_path=f'{config["PROJECT"]["VolumeMountPath"]}',
-        container_disk_in_gb=int(config["PROJECT"]["ContainerDiskSizeGB"]),
-        env=environment_variables
-    )
-
+    
+    selected_gpu_types = map(lambda s: s.strip(),config['PROJECT']['GPU'].split(','))
```

Comment:
What are your thoughts on either creating the key "gpu_preferences" or "gpu_list" and requiring the data type to be a list vs needing to parse a string list?